examples/example-ironoxide-user.json

```diff
@@ -0,0 +1,6 @@
+{
+  "accountId": "example-ironoxide-user",
+  "segmentId": 1825,
+  "signingPrivateKey": "IW67bY+Yee6oLV1T/AlS82bSP7d9ZqDH3EhmRGJj9js06T9Rx2SjAI3FRUf9O7sflOJ61gDpXVrAO/sNfGZoXw==",
+  "devicePrivateKey": "W7DvCJvGsnXunza+qstf/FTLW42h/VpWN8Be/CkClhY="
+}
```

Comment:
Where is this device from?

Reply:
Generated using the production pem file in the tests. We can't comment the json file and keep it json. Do you think it's important to comment on that?

Reply:
No, I don't think we need to comment it. We've just recently been using a specific project for public devices in tests, and we haven't made ironoxide's public yet, so I was wondering where you decided to get it from.